pages/common/c99.md

```diff
@@ -0,0 +1,19 @@
+# C99
+
+> Compiles C programs according to the ISO C standard.
+
+- Compile source files and create an executable:
```

Comment:
Perhaps I should do "file(s)"? Or I could follow the gcc page's example, and do "{{source1.c}} {{source2.c}}"